daprdocs/content/en/reference/components-reference/supported-secret-stores/kubernetes-secret-store.md

```diff
@@ -7,12 +7,30 @@ aliases:
   - "/operations/components/setup-secret-store/supported-secret-stores/kubernetes-secret-store/"
 ---
 
-## Summary
+## Default Kubernetes secret store component
+When Dapr is deployed to a Kubernetes cluster, a secret store with the name `kubernetes` is automatically provisioned. This is meant to streamline the usage of the native Kubernetes secret store but generally, it is a better practice to create a component definition like the one below with a custom name. Using a custom definition decouples referencing the secret store in your code from the hosting platform (Kubernetes) keeping you code more generic and portable. Additionally, by explicitly defining a Kubernetes secret store component you can connect to a Kubernetes secret store from a local Dapr self-hosted installation. This requires a valid [`kubeconfig`](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) file.
 
-Kubernetes has a built-in secrets store which Dapr components can use to retrieve secrets from. No special configuration is needed to setup the Kubernetes secrets store, and you are able to retrieve secrets from the `http://localhost:3500/v1.0/secrets/kubernetes/[my-secret]` URL. See this guide on [referencing secrets]({{< ref component-secrets.md >}}) to retrieve and use the secret with Dapr components.
+When limiting access to secrets in your application using [secret scopes]({{<ref secrets-scopes.md>}}), it's important to remember this store was automatically created and so to include it in the scope definition.
```

Comment:
```suggestion
{{% alert title="Scoping secret store access" color="warning" %}}
When limiting access to secrets in your application using [secret scopes]({{<ref secrets-scopes.md>}}), it's important to include this default secret store in the scope definition in order to restrict it.
{{% /alert %}}
```

Reply:
@orizohar @msfussell @yaron2 is it me or does this seem like a security issue?

Reply:
There's a wording issue here, `When` should be changed to `If`.

Reply:
This is not a security issue, since you still need to be using scopes